sklearn/decomposition/_fastica.py

```diff
@@ -628,6 +639,7 @@ def transform(self, X, copy=True):
         Returns
         -------
         X_new : ndarray of shape (n_samples, n_components)
+            Transformed data obtained with unmixing matrix.
```

Comment:
If you want to speak about sources, maybe you could phrase it as:

Estimated sources obtained by transforming the data with the estimated unmixing matrix.